src/components/starlight/Search.astro

```diff
@@ -47,6 +47,7 @@ const docSearchStrings = getDocSearchStrings(Astro);
 				// work better on localhost, preview URLS.
 				const url = new URL(item.url);
 				if (url.hash === '#overview') url.hash = '';
```

Comment:
Thanks for fixing this @TheOtterlord!

I think we could also have removed this? This was the old version of this when we used `#overview` for the top-of-page anchor, but I don't _think_ it should exist anywhere anymore?

Reply:
aha, sounds good! I'll confirm this and remove it in a small follow-up PR